proto/valve.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package smartcore.bos;
+
+option go_package = "github.com/vanti-dev/sc-bos/pkg/gen";
+
+import "google/protobuf/field_mask.proto";
+import "google/protobuf/timestamp.proto";
+
+service ValveApi {
+  rpc GetValve(GetValveRequest) returns (Valve) {}
+  rpc PullValve(PullValveRequest) returns (stream PullValveResponse) {}
+  rpc UpdateValve(UpdateValveRequest) returns (Valve) {}
+}
+
+message Valve {
+  // flow rate set point e.g. L/s
+  float flow_rate_set_point = 1;
+  // flow rate sensor e.g. L/s
+  // read_only
+  float flow_rate_sensor = 2;
```

Comment:
Docs should be proper english (Capitalised sentences, punctuation, not snake_case) and should explain what the field represents. These aren't any of those things.